### ISSUE TITLE
Support for Matrix URIs

### ISSUE THREAD

**Issue by HenriqueLimas** [mgonto/restangular#968]
It be cool if Restangular gives support for Matrix URIs. Maybe could be created a new Restangular methods "matrixParams" where we can define the matrix URIs. 
### Today:

``` javascript
Restangular.all('cambridge;scale=50000').post();
```
### Purpose:

``` javascript
Restangular.all('cambridge').matrixParams({scale: 5000}).post()
```

**Doc:** http://www.w3.org/DesignIssues/MatrixURIs.html

**Comment by grabbou** [comment]
Hey! Great suggestion, to be supported in V2. If you feel powerful enough - please consider doing a PR for this particular feature :) Should not be that hard, it all boils down to exposing matrixParams() method that can be called any time on requestless Restangular object just to modify current route element.

**Comment by HenriqueLimas** [comment]
Yeah, I am going to make a PR. But, what is the branch I can do that? The "2.0-wip"?

**Comment by grabbou** [comment]
Thanks! Would love to see that feature :) please use wip/cleanup-improvements to fork and submit PR which is actually our development branch for now.

**Comment by pedromarce** [comment]
Hi, I know it has been a long time, but trying to tidy a little bit, would still be interesting? 
Also, wouldn't it make more sense to use it as a configuration flag, so you define parameters same way and based on this parameter the final URL gets built as matrix URI or the existing one?

**Comment by daviesgeek** [comment]
@HenriqueLimas Is this still something you want to add?

**Comment by HenriqueLimas** [comment]
@daviesgeek, no nothing to add.

**Comment by daviesgeek** [comment]
@HenriqueLimas Sorry I should have been more clear. You have a PR up; is this still something that you would like to merge in? I'll work with you to merge it in if necessary.

**Comment by HenriqueLimas** [comment]
I think is something good to have, someone could want that. But I am not using it anymore..if you think is good to have that, we can merge or just close the pull request.

**Comment by daviesgeek** [comment]
Okay. This sounds like a good idea to add this, but I'm not familiar with it at all, so I won't merge right now, but will wait for other collaborators to check it out.
